R/getDominantTaxa.R

```diff
@@ -0,0 +1,152 @@
+#' Get Dominant Taxa
+#'
+#' This function return the most dominant taxa.
+#'
+#' @param x
+#' A \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#' object
+#'
+#' @param rank A single character defining a taxonomic rank. Must be a value of
+#'   \code{taxonomicRanks()} function.
```

Comment:
... of the output of \code{taxonomicRanks()}

Reply:
Changed